play-media-native/components/Listing/Listing.tsx

```diff
@@ -18,6 +19,19 @@ import { Screen } from '../../features/Screen/Screen';
 import { mockFetchData } from '../../helpers/mockPagination';
 import { theme } from '../../theme/theme';
 
+const pageStyles = StyleSheet.create({
+  footerContainer: {
+    height: theme.spacing.xl,
+    marginTop: theme.spacing.sm,
+    marginLeft: theme.spacing.sm,
```

Comment:
please add some padding on the right to ensure that even with longer text it will not be hidden under the button